src/App/Action/Backend/Order/UpdateOrderStatusAction.php

```diff
@@ -47,7 +47,7 @@ public function handle(Request $request): Response
 
         $status = SettingsFacade::get($settingName, OrderSettings::ID);
 
-        if (Settings::OPTION_NONE === (int) $status) {
+        if (Settings::OPTION_NONE === (int) $status || null === $status) {
```

Comment:
Waarom kan dit überhaupt null zijn? In de config kan je ervoor zorgen dat dingen standaard waarden krijgen vanuit installatie. `null` is weer een extra mogelijke waarde die we nu ook weer moeten afhandelen met allemaal enge codes.

Zie `builtInDefaultSettings` in de config. Dit is wss nodig voor alle tristate velden die niet null mogen zijn.